### ISSUE TITLE
Orgrimmar - some NPCs under terrain

### ISSUE THREAD

**Issue by GameMaster0x** [azerothcore/azerothcore-wotlk#20807]
### Current Behaviour

Quite a few NPCs in OG are below the terrain.
![WoWScrnShot_120124_144628](https://github.com/user-attachments/assets/03d5541d-f1e8-48ec-b207-3ec9fd67de7a)
![WoWScrnShot_120124_144653](https://github.com/user-attachments/assets/5930e831-b8db-43df-8af1-30cb6b5a6144)
![WoWScrnShot_120124_144702](https://github.com/user-attachments/assets/9aad3e63-a09c-40fe-a164-5a031c24e9dc)
![WoWScrnShot_120124_144718](https://github.com/user-attachments/assets/d0e731df-d2b3-40bd-b7e7-b3e8bbc7e969)
![WoWScrnShot_120124_144543](https://github.com/user-attachments/assets/30678c47-e198-4643-b36f-9d6962654176)
![WoWScrnShot_120124_144610](https://github.com/user-attachments/assets/45fb028e-35eb-4a5c-9b22-5e7d2fda53fb)

### Expected Blizzlike Behaviour

The shouldn't be there. 

### Steps to reproduce the problem

Go to OG, check.

### AC rev. hash/commit

482f4c94ffdb

### Operating system

Win11 x64

**Comment by GroodeeshPanj** [comment]
Yeah, looks fine to me on a build from an hour ago. Maybe a case can be made for the one toe on the fishing supplies... *shrug*
![blsmth](https://github.com/user-attachments/assets/adc8a6ae-fbfb-4b01-ba4d-725204b00611)
![blsmth2](https://github.com/user-attachments/assets/c3b74f29-d78b-4345-aa8b-a7308e84c310)
![shankys](https://github.com/user-attachments/assets/933c6ed4-e40c-4084-b563-bc7b8acee04a)
![xor-juul](https://github.com/user-attachments/assets/297c1566-8261-4745-bb05-206ff725b049)

**Comment by TheSCREWEDSoftware** [comment]
> Yeah, looks fine to me on a build from an hour ago. Maybe a case can be made for the one toe on the fishing supplies... _shrug_ ![blsmth](https://private-user-images.githubusercontent.com/186625738/391358843-adc8a6ae-fbfb-4b01-ba4d-725204b00611.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODUzMTgsIm5iZiI6MTczMzA4NTAxOCwicGF0aCI6Ii8xODY2MjU3MzgvMzkxMzU4ODQzLWFkYzhhNmFlLWZiZmItNGIwMS1iYTRkLTcyNTIwNGIwMDYxMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwMVQyMDMwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZDM1NjQxNTExZGMxY2MzMzVhMzNhYzRiOTYwNTU0NTZjYWUyMWEzMGZiMzc0OGQyYzAzMmI3N2FmZGExMjJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cOfFQJV4cvOozlxg5Z9DDN81Fljpu-lkrrsk97r5_so) ![blsmth2](https://private-user-images.githubusercontent.com/186625738/391358846-c3b74f29-d78b-4345-aa8b-a7308e84c310.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODUzMTgsIm5iZiI6MTczMzA4NTAxOCwicGF0aCI6Ii8xODY2MjU3MzgvMzkxMzU4ODQ2LWMzYjc0ZjI5LWQ3OGItNDM0NS1hYThiLWE3MzA4ZTg0YzMxMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwMVQyMDMwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZGM1NGM3MDQyYjZmMzIyMzdjZTdmMzc0NTVjZmNlNWZhMWUzMjc1NGNkOGZhODczNjQzZGVkMjM3MjE4ZjJjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.yqzy7Z4KKMM-p02sMoKSiROKTQA7Lx-8gMMaWuDLt4g) ![shankys](https://private-user-images.githubusercontent.com/186625738/391358848-933c6ed4-e40c-4084-b563-bc7b8acee04a.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODUzMTgsIm5iZiI6MTczMzA4NTAxOCwicGF0aCI6Ii8xODY2MjU3MzgvMzkxMzU4ODQ4LTkzM2M2ZWQ0LWU0MGMtNDA4NC1iNTYzLWJjN2I4YWNlZTA0YS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwMVQyMDMwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00YmQ2NWUxOGFkNDY2YTg5ZGI3YTVmZDQwYjFiODM3NTQzNDJjNTc3OGQxNjgyN2E3Yjg0MjU0ZjM3NDczY2E1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.61QkChZW62YgalfHbL7viJcXbVBrqii28IFb1-aSuTY) ![xor-juul](https://private-user-images.githubusercontent.com/186625738/391358849-297c1566-8261-4745-bb05-206ff725b049.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODUzMTgsIm5iZiI6MTczMzA4NTAxOCwicGF0aCI6Ii8xODY2MjU3MzgvMzkxMzU4ODQ5LTI5N2MxNTY2LTgyNjEtNDc0NS1iYjA1LTIwNmZmNzI1YjA0OS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwMVQyMDMwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZWFiYmIwM2JlZDBlM2VkZTUxY2Q0ODM4NjRkNzZhNjdkNGRiZmIxMTIwMGY2ZWJmMjAwYjAzM2ExYmY0NWRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.02pKlJuIvdKLQ5Nx3jjwDBefsttSc1oBx8WScg4spmU)

Very minimal but can be fixed, not much of a hassle. Can you provide the GUID of the fisher? (a screenshot is fine)

**Comment by Hynisel** [comment]
Hello, I have found a similar problem in ice crown outside the tournament, it is something small but would be nice if it could be fixed. As is possible to see from the screenshot, the npc Eydis Darkbane is half under the ground.

![1](https://github.com/user-attachments/assets/01aa76a1-f54a-4a77-8ca9-4177413213c2)

**Comment by TheSCREWEDSoftware** [comment]
Just checked the fisher trainer. She's in the correct position it's just the object hides her foot (place yourself in her position it does the same for you).

@Hynisel i will submit my PR that fixes her.
